org-cyf-itd/content/blocks/step-1-using-slack/instructions/index.md

```diff
@@ -0,0 +1,30 @@
++++
+title ="Step 1: What to do"
+description= "Essential Digital Skills: Navigating Slack & Google Search"
+emoji= "🗨"
+time= 60
+[build]
+  render = 'never'
+  list = 'local'
+  publishResources = false 
++++
+
+### Essential Digital Skills: Navigating Slack & Google Search
+
+#### Overview
+
+**Slack** is a professional communication and collaboration tool widely used across industries, especially in tech.
+
+**Search engines** (such as Google Search) are powerful tools that IT professionals rely on to quickly find solutions and troubleshoot issues.
+
+This exercise aims to help you 
+- Become familiar with Slack so you can use it effectively in CodeYourFuture's courses and also in a professional setting
+- Enhance self-sufficiency in finding solutions using search engines
+
+#### Steps
+
+1. Answer the questions on [Slack and Search Engine Exercise](https://docs.google.com/forms/d/e/1FAIpQLSc-xeV91g0TSWtbDVcYIxJ9AJRo_PJYzdd9AaKXJBKHt6sihA/viewform?usp=dialog)
```

Comment:
I'm a little unsure on the "domain name" question - I'd generally say `itd.codeyourfuture.io` is a domain name - shall we maybe use `https://itd.codeyourfuture.io` as the other misleading answer rather than `itd.codeyourfuture.io`?